cmd/utils/flags.go

```diff
@@ -558,6 +558,28 @@ var (
 		Value:    ethconfig.Defaults.Miner.NewPayloadTimeout,
 		Category: flags.MinerCategory,
 	}
+	MevEnabledFlag = &cli.BoolFlag{
```

Comment:
We should make the flag names and description more clear.

1. Add a universal prefix, e.g. `mev`.
2. `sequencer.url` is not accurate, the forward url is not necessary to be a sequencer.
3. pricefloor is better to be bundle gas price floor.
4. `mev.enable` is better to be `mev.miner.enable`.

Reply:
Yes, make sense. I think we just need `mev.enalbe` flag to enable bundlepool, `mev.bundle.receiver.url` flag to decide whether to deliver bundle and `mev.bundle.gasprice.floor` to be bundle gas price floor. Please review again.